packages/venia-concept/src/components/CreateAccountPage/helpers.js

```diff
@@ -0,0 +1,9 @@
+const validCreateAccountParams = ['email', 'firstName', 'lastName'];
+
+export const getCreateAccountInitialValues = () => {
+    const params = new URLSearchParams(window.location.search);
+    return validCreateAccountParams.reduce(
+        (values, param) => ({ ...values, [param]: params.get(param) }),
+        {}
+    );
+};
```

Comment:
This would be a good file to have tests for, since it has edges. Consider at least verifying that it doesn't throw errors in common cases.

Reply:
I agree